tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -750,16 +750,20 @@ public function test_process_directives_does_not_change_inner_html_in_math() {
 	 */
 	private function evaluate( $directive_value ) {
 		$generate_state = function ( $name ) {
+			$obj       = new stdClass();
+			$obj->prop = $name;
 			return array(
 				'key'    => $name,
 				'nested' => array( 'key' => $name . '-nested' ),
+				'obj'    => $obj,
 			);
```

Comment:
We can also test array access:
```suggestion
				'arrAccess' => new class() implements ArrayAccess {
					public function offsetExists($offset): bool { return true; }
					public function offsetGet($offset): mixed { return $offset; }
					public function offsetSet($offset, $value): void {}
					public function offsetUnset($offset): void {}
				},
			);
```